### PR TITLE
Fixing bugs from SonarCloud

### PR DESCRIPTION
**Pull request by Kamil4921**
Html code from small issues eg: adding rel or <ul>
Issue https://github.com/DevAdventCalendar/DevAdventCalendar/issues/328

### CODE REVIEW

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Account/ExternalLoginFailure.cshtml [hunk 1]
```diff
@@ -1,8 +1,7 @@
-﻿@{
+@{
     ViewBag.Title = "Błąd logowania";
 }
 
-<hgroup>
+
```
Comment (WTobor) [inline src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Account/ExternalLoginFailure.cshtml:5]: have you checked if it looks ok?

Comment (Kamil4921) [reply src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Account/ExternalLoginFailure.cshtml:5]: Sorry perhaps I should use header tag. Am I right?

Comment (Kamil4921) [reply src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Account/ExternalLoginFailure.cshtml]: I replaced it by <div> tag. Is it correct now?

Comment (WTobor) [reply src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Account/ExternalLoginFailure.cshtml]: I have no idea :p Have checked the view?

Comment (WTobor) [reply src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Account/ExternalLoginFailure.cshtml]: @Kamil4921 ?

**Pull request merged**